### DISCUSSION TITLE
Mushroom Chip card - Moon entity language

### DISCUSSION THREAD

**Discussion by misabohac** [piitaya/lovelace-mushroom#927] [General]
Hi guys, after few years I had to reinstall and reconfigure my whole Home Assistant system running on VM in Synology.
Now I have problem with mushroom chip card - moon entity.
In the past it was showing correctly the moon phase in czech language.
Now in the chip card it is showing the moon phase in english.
If I add lovelace entity card for the moon phase, it is shown in czech language correctly.

Any idea, how to fix this issue to show the attributes in czech language?

Thanks a lot!

**Comment by piitaya** [comment]
There is a change for some translations in HA `2023.1`. Here's the PR with the fix #928

**Reply by misabohac** [reply]
Hello, thanks a lot for your fast answer.
Means, that this bug will be fixed soon in some update?

Thank you!

**Reply by piitaya** [reply]
Yep

**Reply by misabohac** [reply]
Great, thanks a lot for letting us know man.
Have a nice day!
